P5/Source/Specs/att.scoping.xml

```diff
@@ -38,8 +38,8 @@ $Id$
         </egXML>
       </exemplum>
 
-      <remarks versionDate="2016-03-15" xml:lang="en">
-        <p>The expression of certainty applies to the nodeset identified by the value of the
+      <remarks versionDate="2021-01-30" xml:lang="en">
+        <p>The expression applies to the nodeset identified by the value of the
```

Comment:
I am uncomfortable with this. “The expression” seems ambiguous. I would prefer something like “The semantics of the element upon which this attribute appears apply to the …”.

Reply:
@sydb If I'm moving these remarks so they're a child of classSpec, it won't make sense to be discussing just the `@match` attribute, but rather the whole element. I'm trying just this (slightly simpler) phrasing: 
`The semantics of this element apply to the nodeset identified by the value of the
      <att>target</att> attribute, possibly modified additionally by the value of the
      <att>match</att> attribute.`

Reply:
@sydb I totally agree about simplifying the `desc` for `@target`...implementing that.